stan/math/opencl/opencl_context.hpp

```diff
@@ -195,6 +195,11 @@ class opencl_context_base {
     int cholesky_rev_block_partition = 8;
     // used in math/opencl/multiply
     int multiply_split_upper_limit = 2000000;
+    // used in math/prim/mat/fun/gp_exp_quad_cov
+    int gp_exp_quad_cov_size = 466100;
+    // TODO(Tadej): next two parameters need better tunning
```

Comment:
Hmm. Maybe I could make this variable name more informative. It is the size of resulting matrix at wich it is worth to use GPU instead of CPU. It is used in versions that accept one or two `std::vector<double>` arguments - earlier in this thread i called them 1 and 2.